BlueDBOnDisk/src/main/java/org/bluedb/disk/segment/rollup/RollupScheduler.java

```diff
@@ -82,6 +82,17 @@ public boolean isRunning() {
 		return !isStopped;
 	}
 
+	protected void scheduleLimitedReadyRollups() {
+		int rollupsToSchedule = 30 - collection.getQueuedTaskCount();
+		scheduleReadyRollups(rollupsToSchedule);
+	}
+
+	protected void scheduleReadyRollups(int maxRollupsToSchedule) {
+		for (RollupTarget target: rollupTargetsReadyForRollup(maxRollupsToSchedule)) {
+			scheduleRollup(target);
+		}
+	}
+
 	protected void scheduleReadyRollups() {

```

Comment:
Is this method still called? It could be dangerous to call it if we're in a big collection that hasn't been rolling up correctly